lib/lyber_core/workflow_client_factory.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module LyberCore
+  # Factory for creating a workflow client
+  class WorkflowClientFactory
+    def self.build
+      logger = Logger.new(Settings.workflow.logfile, Settings.workflow.shift_age)
+      Dor::Workflow::Client.new(url: Settings.workflow.url, logger: logger, timeout: Settings.workflow.timeout)
```

Comment:
Does this work even without the gem declaring the `config` gem as a dependency? Maybe it's never a problem in a real world since presumably all of lyber-core's dependents will be using the `config` gem themselves? Like, does this grab the dependent's configs or do something weird? I ask because we typically do not use the `config` gem in gems so this is a bit uncharted.

Reply:
Good point. I'm going to declare as dependency. I'm going to be moving the common `config/boot.rb` code into this gem as well, which will take care of setting up config.

Reply:
An alternative to using the `config` gem in a gem that we may want to consider, which we use in our other gems, is adding a `.configure` method to the gem's public interface which lets consumers inject the values they need.